vault/proxy.go

```diff
@@ -0,0 +1,296 @@
+package vault
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	socks5 "github.com/armon/go-socks5"
+	isatty "github.com/mattn/go-isatty"
```

Comment:
you shouldn't need the aliases here; IIRC, golang automagically strips the `go-` prefix from any import terminal names.

Reply:
goimports does this automatically. Gonna leave this to avoid jitter in the future.

Reply:
dumb, but okay 😄